File: Modul 1/PRAK110-2410817110020-AkmalRizki.c
#include<stdio.h>
int main()
{
    float alas, tinggi, miring, keliling, luas;
    alas=5;
    tinggi=12;
    miring=(alas*alas+tinggi*tinggi);
    keliling=alas+tinggi+miring;
    luas=(alas*tinggi)/2;
    printf("diketahui :\n");
    printf("alas = %.0f cm\n",alas);
    printf("tinggi = %.0f cm\n\n",tinggi);
    printf("jawaban :\n");
    printf("sisi A = %0.f cm \n", tinggi);
    printf("sisi B = %0.f cm \n", miring);
    printf("sisi C = %0.f cm \n", alas);
    printf("keliling = %0.f cm \n", keliling);
    printf("luas = %0.f cm", luas);
}

File: Modul 1/PRAK110-2410817110020-AkmalRizki.py
alas=5
tinggi=12
miring=(alas*alas+tinggi*tinggi)**0.5
keliling=alas+tinggi+miring
luas=(alas*tinggi)/2
print(f"diketahui :")
print(f"alas = {alas:.0f} cm")
print(f"tinggi = {tinggi:.0f} cm\n")
print(f"jawaban :")
print(f"sisi A = {tinggi:.0f} cm")
print(f"sisi B = {miring:.0f} cm")
print(f"sisi C = {alas:.0f} cm")
print(f"keliling = {keliling:.0f} cm")
print(f"luas = {luas:.0f} cm")